src/main/java/com/refinedmods/refinedstorage/apiimpl/autocrafting/task/v6/CraftingTask.java

```diff
@@ -596,6 +586,19 @@ private ICraftingTaskError calculateInternal(
         return null;
     }
 
+    @Override
+    public void start() {
+        crafts.values().forEach(craft -> {
```

Comment:
Isn't the missing.isEmpty step needed?

Reply:
hmm it's not needed but probably a good idea to put one there.